src/test/fuzz/p2p_headers_presync.cpp

```diff
@@ -197,15 +197,15 @@ FUZZ_TARGET(p2p_headers_presync, .init = initialize)
                 auto headers_msg = NetMsg::Make(NetMsgType::BLOCK, TX_WITH_WITNESS(block));
                 g_testing_setup->SendMessage(fuzzed_data_provider, std::move(headers_msg));
             });
+    }
 
-        // This is a conservative overestimate, as base is only moved forward when sending headers. In theory,
-        // the longest chain generated by this test is 1600 (FUZZ_MAX_HEADERS_RESULTS * 100) headers. In that case,
-        // this variable will accurately reflect the chain's total work.
-        total_work += CalculateClaimedHeadersWork(all_headers);
+    // This is a conservative overestimate, as base is only moved forward when sending headers. In theory,
+    // the longest chain generated by this test is 1600 (FUZZ_MAX_HEADERS_RESULTS * 100) headers. In that case,
+    // this variable will accurately reflect the chain's total work.
+    total_work += CalculateClaimedHeadersWork(all_headers);
 
-        // This test should never create a chain with more work than MinimumChainWork.
-        assert(total_work < chainman.MinimumChainWork());
-    }
+    // This test should never create a chain with more work than MinimumChainWork.
+    assert(total_work < chainman.MinimumChainWork());
```

Comment:
nit: You could get rid of `total_work` as a variable and just do:

```suggestion
    assert(CalculateClaimedHeadersWork(all_headers) < chainman.MinimumChainWork());
```

Reply:
If I add the first block to `all_headers` then I could do this. I do like the variable for clarity, and I'd say this assertion is mainly to make the test easier to read. So I'll keep it as is.